packages/sdk/src/base/directus.ts

```diff
@@ -159,7 +156,18 @@ export class Directus<T extends TypeMap> implements IDirectus<T> {
 		);
 	}
 
+	collections<C extends string, I = TypeOf<T, C>>(collection: C): CollectionsHandler<I> {
+		return (
+			this._collections[collection] ||
+			(this._collections[collection] = new CollectionsHandler<T>(collection, this.transport))
+		);
+	}
```

Comment:
Does this have the same problem as fields/relations where there is a root endpoint that doesn't use the collection scoping? (https://docs.directus.io/reference/api/system/collections/#list-collections)